docs/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="refresh" content="0; url=https://heremaps.github.io/harp.gl/doc">
```

Comment:
Does this have to be an absolute URL?

Reply:
I guess not, but is there any good reason to change it to a relative URL?